components/openindiana/time-slider/Makefile

```diff
@@ -21,7 +21,7 @@ COMPONENT_VERSION=	0.2.101
 COMPONENT_LICENSE=	CDDL
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count).1
```

Comment:
And the republishing needs the version increment.  Both in the component and in the userland-incorporation.  For illumos-gate it is not a big deal since the gate changes very often.  If we do some packaging changes for gfx-drm (outside the gfx-drm gate) we should increment the gfx-drm version too.

But as I understood you hates incorporations, so I'll remove the version change.  Better to have this fixed with the same version number, than not fixed at all.

And, there is nothing strange with time-slider@0.2.101-2017.0.0.107.1 :-).